datasets/new-dataset-using-llm/README.md

```diff
@@ -0,0 +1,79 @@
+# Synthetic Data Generation
```

Comment:
And better to include the license info as well.

Reply:
Add openai and phi license link to the note